Open-ILS/src/templates/staff/navbar.tt2

```diff
@@ -14,144 +14,143 @@
 
   <!-- navbar-header here needed for supporting angular-ui-bootstrap -->
   <div class="navbar-header">
-    <button type="button" class="navbar-toggle" 
+    <button type="button" class="navbar-toggle"
         ng-init="navCollapsed = true" ng-click="navCollapsed = !navCollapsed">
       <span class="sr-only">[% l('Toggle navigation') %]</span>
       <span class="icon-bar"></span>
       <span class="icon-bar"></span>
-      <span class="icon-bar"></span>
-    </button>
-  </div>
+       span class="icon-bar"></span>
```

Comment:
This seems to lose an opening angle bracket (<) for the span tag, and for the button tag on the next line.